integrationTests/frontend/wallet/txInterception_test.go

```diff
@@ -22,13 +23,14 @@ func TestInterceptedTxWithoutDataField(t *testing.T) {
 		0,
 		value,
 		"erd12dnfhej64s6c56ka369gkyj3hwv5ms0y5rxgsk2k7hkd2vuk7rvqxkalsa",
-		"erd15xa6lw0g8uvnkl82p9p9ks5guvp02twuwky4m936lhtup4cppejsg2rv0g",
-		"9f0e3bccbe1815580e2f8db150405e2f88255aaf801cde204af8728ccd4d43d1ab82188a24050e00d1cc21f85eab251014e16984774957ce06d2060fd77e9f0f",
+		"erd1l20m7kzfht5rhdnd4zvqr82egk7m4nvv3zk06yw82zqmrt9kf0zsf9esqq",
```

Comment:
because I had to update the version so I had to regenerate the transactions' data. old version still works but not with the updated `MinTransactionVersion` from integrationTests package